lib/backends/UHT-Contacts/index.js

```diff
@@ -65,6 +65,14 @@ async function fetchCustomer(id) {
   ]))[0];
 }
 
+async function fetchActionDiariesNotes(id) {
```

Comment:
Sorry - one more minor nitpick - We are fetching notes from one action diary, so this should be `fetchActionDiaryNotes`